src/interactive.js

```diff
@@ -1,75 +1,183 @@
+              <p class="todo-text" contenteditable="true">${task.todoText}</p><div class="del-menu"><a class="edit" href="#">
+              <i class="fas fa-edit edit-box"></i></a><a href="#" class="del"><i class="fas fa-trash del-box"></i></a></div></div>`;
           li.classList.add('list-item');
           li.innerHTML = text;
           list.appendChild(li);
         }
       });
+      const check = document.querySelectorAll('.check-input');
+      const todos = JSON.parse(localStorage.getItem('todos'));
+      check.forEach((check, index) => {
+        check.addEventListener('change', (e) => {
+          e.preventDefault();
+          const divWrapper = check.parentElement;
+          if (check.checked) {
+            divWrapper.classList.add('checked');
+            check.checked = true;
+            todos[index].completed = true;
+            localStorage.setItem('todos', JSON.stringify(todos));
+          } else {
+            divWrapper.classList.remove('checked');
+            todos[index].completed = false;
+            localStorage.setItem('todos', JSON.stringify(todos));
+          }
+        });
+        window.addEventListener('load', (e) => {
+          e.preventDefault();
+          const divWrapper = check.parentElement;
+          if (check.checked) {
+            divWrapper.classList.add('checked');
+            check.checked = true;
+            todos[index].completed = true;
+            localStorage.setItem('todos', JSON.stringify(todos));
+          } else {
+            divWrapper.classList.remove('checked');
+            todos[index].completed = false;
+            localStorage.setItem('todos', JSON.stringify(todos));
+          }
+        });
+      });
     }
+  }
 
-    const check = document.querySelectorAll('.check-input');
-    const todos = JSON.parse(localStorage.getItem('todos'));
-    check.forEach((check, index) => {
-      check.addEventListener('change', (e) => {
+  editItem() {
+    const edit = document.querySelectorAll('.edit');
+    const todoList = JSON.parse(localStorage.getItem('todos'));
+    edit.forEach((v, i) => {
+      v.addEventListener('click', (e) => {
         e.preventDefault();
-        const divWrapper = check.parentElement;
-        if (check.checked) {
-          divWrapper.classList.add('checked');
-          check.checked = true;
-          todos[index].completed = true;
-          localStorage.setItem('todos', JSON.stringify(todos));
-        } else {
-          divWrapper.classList.remove('checked');
-          todos[index].completed = false;
-          localStorage.setItem('todos', JSON.stringify(todos));
-        }
+        const aDiv = v.parentElement;
+        const listItem = aDiv.parentElement;
+        const p = listItem.children[1];
+
+        todoList[i].todoText = p.innerText;
+        localStorage.setItem('todos', JSON.stringify(todoList));
       });
+    });
+  }
+
+  delItem() {
```

Comment:
- [ ] This function is not working properly, sometimes it removes the correct task and sometime it does not, please fix this issue 😄